src/set_theory/ordinal/arithmetic.lean

```diff
@@ -648,7 +648,7 @@ by rw [mul_add, mul_one]
 theorem mul_two (a : ordinal) : a * 2 = a + a :=
 by { change a * (succ 1) = a + a, rw [mul_succ, mul_one] }
 
-instance mul_covariant_class : covariant_class ordinal.{u} ordinal.{u} (*) (≤) :=
+instance has_le.le.mul_covariant_class : covariant_class ordinal.{u} ordinal.{u} (*) (≤) :=
```

Comment:
```suggestion
instance covariant_class_mul_le : covariant_class ordinal.{u} ordinal.{u} (*) (≤) :=
```

Reply:
I named this for consistency with other existing instances. Might as well rename those too.

Reply:
What do you mean? `covariant_class_op_rel` is the naming convention.

Reply:
We previously had `ordinal.has_le.le.add_contravariant_class`, among others. I've renamed them according to your suggestion.